app/components/avo/actions_component.rb

```diff
@@ -2,40 +2,47 @@
 
 class Avo::ActionsComponent < Avo::BaseComponent
   include Avo::ApplicationHelper
-  attr_reader :label, :size, :as_row_control
-
-  def initialize(actions: [], resource: nil, view: nil, exclude: [], include: [], style: :outline, color: :primary, label: nil, size: :md, as_row_control: false, icon: nil)
-    @actions = actions || []
-    @resource = resource
-    @view = view
-    @exclude = Array(exclude)
-    @include = include
-    @color = color
-    @style = style
-    @label = label || I18n.t("avo.actions")
-    @size = size
-    @icon = icon
-    @as_row_control = as_row_control
+
+  ACTION_FILTER = _Set(_Class(Avo::BaseAction))
```

Comment:
I don’t know if this type is too precise for your taste. It says it must be a Set of classes that inherit from `Avo::BaseAction`. I find the documentation value of spelling this out to be helpful, but we could also just use `Set` as the type.

Reply:
This is a good enforcement! We have that assumption that we expect BaseAction descendents.